.github/workflows/build-runner-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Runner EC2 Images
+
+on:
+  schedule:
+    # 00:00 on Monday each week
+    - cron: "0 0 * * 1"
+    
+jobs:
+  build-image:
+    name: Build
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packer/github-actions
```

Comment:
Maybe call the directory `github-actions-runner` to make it extra clear what it's for.